relay-protocol-derive/src/lib.rs

```diff
@@ -541,6 +592,13 @@ fn derive_metastructure(mut s: synstructure::Structure<'_>, t: Trait) -> syn::Re
                         __child_meta
                     }
                 }
+
+                #[automatically_derived]
+                gen impl ::relay_protocol::IntoValueObject for @Self {
+                    fn into_object_fields(self) -> ::relay_protocol::Object<::relay_protocol::Value> {
+                        todo!();
```

Comment:
Is this expected to be here?

Reply:
Sorry, that was a big oversight, that's what I get from working on a follow up branch instead of that one, this is addressed now.